docs/ldml/tr35-keyboards.md

```diff
@@ -420,7 +415,8 @@ Element used to keep track of the source data version.
 
 _Attribute:_ `number` (required)
 
-> The data revision version. The attribute value must start with `$Revision` and end with `$`.
+> Must be a [semver-compatible](https://semver.org) version number.
+<!-- TODO make the above into a reverence to SEMVER -->
```

Comment:
Minor nit:  this TODO is addressed now, isn't it?  Or do we want to expound on this reference further?

Reply:
It's an editorial note. Unicode has a certain structure for making stable references to external standards. Just reminding myself so I don't forget.